Apps/WebClient/src/NewClientApp/src/components/public/vaccine-card/VaccineCardView.vue

```diff
@@ -0,0 +1,638 @@
+                        </v-col>
+                    </v-row>
+                    <HgButtonComponent
+                        id="privacy-statement"
+                        aria-label="Privacy Statement"
+                        variant="link"
+                        data-testid="btnPrivacyStatement"
+                        class="shadow-none pa-0 mt-4"
+                        prepend-icon="info-circle"
+                    >
+                        Privacy Statement
+                        <v-overlay
+                            activator="parent"
+                            location-strategy="connected"
+                            scroll-strategy="block"
+                        >
+                            <v-card class="pa-2" width="250">
+                                <span class="text-body-2">
+                                    Your information is being collected to
+                                    provide you with your COVID‑19 vaccination
+                                    status under s. 26(c) of the
+                                    <em
+                                        >Freedom of Information and Protection
+                                        of Privacy Act</em
+                                    >. Contact the Ministry Privacy Officer at
+                                    <a
+                                        href="mailto:MOH.Privacy.Officer@gov.bc.ca"
+                                        >MOH.Privacy.Officer@gov.bc.ca</a
+                                    >
+                                    if you have any questions about this
+                                    collection.
+                                </span></v-card
+                            >
+                        </v-overlay>
+                    </HgButtonComponent>
+                    <div class="text-center">
+                        <v-row class="my-4 no-gutters d-flex align-center">
+                            <v-col>
+                                <hr />
+                            </v-col>
+                            <v-col cols="auto">
+                                <h3 class="h5 ma-0 px-4 text-muted">OR</h3>
+                            </v-col>
+                            <v-col>
+                                <hr />
+                            </v-col>
+                        </v-row>
+                        <p>Already a Health Gateway user?</p>
+                        <router-link to="/login">
+                            <HgButtonComponent
+                                aria-label="BC Services Card Login"
+                                data-testid="btnLogin"
+                                variant="primary"
+                                class="login-button"
+                                text="Log in with BC Services Card"
+                            >
+                                <span>Log in with BC Services Card</span>
+                            </HgButtonComponent>
```

Comment:
when you set the text, you no longer need to add the value into the default slot
```suggestion
                            />
```